telegram/forcereply.py

```diff
@@ -49,3 +49,5 @@ def __init__(self, force_reply=True, selective=False, **kwargs):
         self.force_reply = bool(force_reply)
         # Optionals
         self.selective = bool(selective)
+
+        self._id_attrs = (self.force_reply, self.selective)
```

Comment:
Adding `selective` to `_id_attrs` differs from the other classes with required args, where no optionals where added. However, `selective` is quite a difference `ForceReply`, actually the only relevant setting (as `force_reply` is always `True`, i.e. the API docs state its Type as `True`, not `Boolean`)